src/control-plane/backend/lambda/api/service/quicksight/sql-builder.ts

```diff
@@ -72,6 +72,24 @@ export interface SQLParameters {
   readonly pairEventAndConditions?: PairEventAndCondition[];
 }
 
+export const buildInEvents = [
```

Comment:
`builtIn` is better

Reply:
@zhu-xiaowei should we simplfy treat the built-in properties starting with `_`?

Reply:
customer may also use '_' as event name first letter.

Reply:
> @zhu-xiaowei should we simplfy treat the built-in properties starting with `_`?

Listing all preset events is more rigorous. We do not restrict users from using events starting with `_`